components/navbar/sidebar.snip.html

```diff
@@ -17,15 +17,15 @@
 }}
 
 <!-- Start Sidebar -->
-<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}}" layout="nodisplay">
+<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} {{sidebarclasses}}" layout="nodisplay">
```

Comment:
Sorry I don't fully understand. In previous PRs you mentioned I should try and use BassCSS classes as much as possible which is the reason for adding the center class to this.

I could add more CSS to the page by adding a center style to ampstart-sidebar but I thought the whole point of using BassCSS was to reduce the amount of custom CSS and therefore less CSS on the page?

In most cases it will always be easier to override ampstart-classes but this will lead to larger CSS.